generate.ts

```diff
@@ -34,7 +34,7 @@ var sdkGeneratorGlobals: SdkGlobals = {
 };
 
 const defaultApiSpecFilePath = "../API_Specs"; // Relative path to Generate.js
-const defaultApiSpecGitHubUrl = "https://raw.githubusercontent.com/PlayFab/API_Specs/master";
+const defaultApiSpecGitHubUrl = "https://raw.githubusercontent.com/PlayFab/API_Specs/TOC";
 const defaultApiSpecPlayFabUrl = "https://www.playfabapi.com/apispec";
 const tocFilename = "TOC.json";
```

Comment:
>tocFilename [](start = 6, length = 11)

May i suggest to also add a constant tocCacheKey = "TOC" here so it's defined only in one place? :)  It will make it easier to change/replace (if ever needed) and also ensure that someone won't write it incorrectly somewhere in the code as a literal

Reply:
Yeah, agreed.